pages/blog/a_straight_guide_to_apis_and_architecture_concepts.md

```diff
@@ -10,35 +10,35 @@ cover: /img/posts/a_straight_guide_to_apis_and_architecture_concepts/cover.webp
 authors:
   - name: Barbaño González
     photo: /img/avatars/barbano.webp
-    link: https://www.linkedin.com/in/barbano-gonzalez-moreno
-excerpt: Basic concepts about APIs and architectures for beginners and non-developers.
+    link: https://www.linkedin.com/in/barbano-gonzalez-moreno 
+excerpt: Basic concepts about APIs and architectures for beginners and non-developers. 
```

Comment:
Why are you submitting a blog post that cites @Barbanio as the author?

You must update the meta data to include your own information and picture.